ide/xml.catalog.ui/src/org/netbeans/modules/xml/catalog/CatalogRootNode.java

```diff
@@ -172,7 +172,7 @@ public HelpCtx getHelpCtx() {
     private static class RootChildren extends Children.Keys implements Comparator, PropertyChangeListener {
         
         /** Contains CatalogReader instances. */
-        private final TreeSet keys = new TreeSet(this);
+        private final Set keys = new TreeSet<>(this);
```

Comment:
I think this can be `Set<CatalogReader>` as the comment states if the item is cast to `CatalogReader` further down in `createKeys(..)`.

Reply:
Done..